docs/SETUP.md

```diff
@@ -1,75 +1,106 @@
 ## Navigation
 
-* [Install](#install)
-* [Configure](#configure)
+* [Requirements](#requirements)
+* [Installation](#installation)
 * [Run](#run)
-    * [Splunk Enterprise](#splunk-enterprise)
-    * [Splunk Universal Forwarder](#splunk-universal-forwarder)
-* [Summary](#summary)
-
-## Install
-In order to run this Docker image, you will need the following prerequisites and dependencies installed on each node you plan on deploying the container:
-1. Linux-based operating system (Debian, CentOS, etc.)
-2. Chipset: 
+    * [Standalone deployment](#standalone-deployment)
+    * [Distributed deployment](#distributed-deployment)
+* [See also](#see-also)
+
+## Requirements
```

Comment:
I noticed that `SUPPORT.md(#system-requirements)` and this section have the same content. I'm thinking we probably want to keep this in one location, but I'm not sure what makes the most sense. Maybe keep it here and have `SUPPORT.md` reference `SETUP.md#requirements`?

Reply:
Yeah... it's entirely redundant. I updated the wording in both places because I hadn't decided where it should go 😆